tensorboard/uploader/proto/export_service.proto

```diff
@@ -13,6 +13,11 @@ service TensorBoardExporterService {
   // Stream the experiment_id of all the experiments owned by the caller.
   rpc StreamExperiments(StreamExperimentsRequest)
       returns (stream StreamExperimentsResponse) {}
+  // Get the metadata of an experiment, including the creation time, name,
+  // description, counts of scalars, tensors and blob sequences, etc. See the
+  // documentation string of `Experiment` for more details.
+  rpc GetExperimentMetadata(GetExperimentMetadataRequest)
+      returns (GetExperimentMetadataResponse) {}
```

Comment:
Per <https://aip.dev/131> and previous discussions, can we just make
this `GetExperiment`, since it returns an `Experiment`?

Reply:
The spec says that

> The response message **must** be the resource itself. (There is no
> `GetBookResponse`.)

—but it looks like this patch still returns `GetExperimentResponse`
rather than `Experiment`?

Reply:
Oops. My bad. Didn't read the AIP spec carefully. Done.